libs/packages/sam-formly/src/lib/formly/types/checkbox.ts

```diff
@@ -18,9 +18,16 @@ import { FieldType } from '@ngx-formly/core';
         <span *ngIf="!to.required && !to.hideOptional"> (Optional)</span>
       </label>
     </div>
-    <div *ngIf="to.tooltipText" class="sds-stack margin-top-205 margin-left-1" [sdsPopover]="to.tooltipText" [sdsPopoverTitle]="to.tooltipTitle" [position]="to.tooltipPosition ? to.tooltipPosition :'right'" tabindex="0" aria-label="info tooltip">
-        <usa-icon [size]="'lg'" [icon]="'info-circle-fill'"></usa-icon>
-    </div>
+
+    <div *ngIf="to.tooltipText" class="padding-top-3 margin-left-1"
```

Comment:
Can we do margins instead of padding on these divs for the extra spacing? Padding makes it such that when the icon is in focus, the focus box occupies the extra whitespace as well vs with margins, the focus border will only be around icon
<img width="638" alt="Screen Shot 2021-09-23 at 2 12 50 PM" src="https://user-images.githubusercontent.com/73653244/134561413-e4e4c6e9-aaed-4909-8fd8-38a58434e3a5.png">
s